Source/Widgets/Viewer/Viewer.js

```diff
@@ -176,6 +180,55 @@ define([
         });
     }
 
+    function enableVRUI(viewer, enabled) {
+        var geocoder = viewer._geocoder;
+        var homeButton = viewer._homeButton;
+        var sceneModePicker = viewer._sceneModePicker;
+        var baseLayerPicker = viewer._baseLayerPicker;
+        var animation = viewer._animation;
+        var timeline = viewer._timeline;
+        var fullscreenButton = viewer._fullscreenButton;
+        var infoBox = viewer._infoBox;
+        var selectionIndicator = viewer._selectionIndicator;
+
+        var display = enabled ? 'none' : 'block';
```

Comment:
Actually I'm having a lot of trouble with this section.  Stomping on all these style rules, in the code that follows this line, can cause problems because this code doesn't really "own" all of these element styles.  I feel like there should be a knockout toggle for the whole UI to support this, but it would have to be more carefully integrated, at least for widgets that already track their own visibility state like the selectionIndicator and infoBox.  @mramato thoughts?